### DISCUSSION TITLE
All the windows binaries are unavailable, the file share site is down or missing, nothing is up, not even chocolately can find them.

### DISCUSSION THREAD

**Discussion by rtmann** [esnet/iperf#1794] [General]
https://files.budman.pw/

Errors out, can't get any of the windows binaries.

**Comment by davidBar-On** [comment]
I don't know what is the problem, but you can get to the binaries through a link shown in [https://www.neowin.net/forum/tags/iperf3/](https://www.neowin.net/forum/tags/iperf3/).
